sc/src/test/java/uk/ac/ebi/atlas/search/geneids/GeneIdSearchDaoIT.java

```diff
@@ -45,6 +53,26 @@
 
     private GeneIdSearchDao subject;
 
+    @BeforeAll
+    void populateDatabaseTables() {
+        ResourceDatabasePopulator populator = new ResourceDatabasePopulator();
+        populator.addScripts(
+//                new ClassPathResource("fixtures/scxa_experiment-fixture.sql"),
+                new ClassPathResource("fixtures/scxa_analytics-fixture.sql"));
+//                new ClassPathResource("fixtures/scxa_marker_genes-fixture.sql"));
```

Comment:
```suggestion
```

Reply:
Although in this case it's a bit silly because you'd end up with 4 separate commits for each changed line, I think. Unless you can do it in bulk but I can't figure out how...